roles/magnum/vars/main.yml

```diff
@@ -81,7 +81,6 @@ _magnum_helm_values:
 _magnum_registry_ingress_annotations:
   # NOTE(mnaser): We only want to allow GET/HEAD requests to the registry
   #               to make sure it's read-only.
-  nginx.ingress.kubernetes.io/configuration-snippet: |
-    if ($request_method !~* "^(GET|HEAD)$") {
-      return 403;
-    }
```

Comment:
This isn't the same thing because this is for CORS only, however, we have actually enabled the registry itself to be read only with another change, so you can remove this whole part because in the environment variables you can see how we set things to be read on

Reply:
okay, removed